input/europe/eu.csv

```diff
@@ -0,0 +1,150 @@
+Country,Name,Reactor no.,Reactor,Status,Capacity in MW,Construction start date,Commercial operation date,Closure (planned),,
```

Comment:
It is a wikipedia page table turned into a csv file:

[wikipedia page](https://en.wikipedia.org/wiki/List_of_nuclear_reactors)
[wikitable to csv](http://wikitable2csv.ggor.de)

I wasn't really sure how i'd script it but I'll edit the readme accordingly!